src/components/VolumeChart/VolumeChart.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import axios from "axios";
+import Chart from "chart.js/auto";
+import { Bar } from "react-chartjs-2";
+import wordedDate from "utils/wordedDate";
+import { StyledVolumeChart } from './VolumeChart.styles'
```

Comment:
relative should come after absolute imports

Reply:
I thought library imports didn't follow that rule.